packages/runtime/container-runtime/src/summaryFormat.ts

```diff
@@ -83,6 +83,12 @@ export interface IContainerRuntimeMetadata {
     readonly disableIsolatedChannels?: true;
     /** 0 to disable GC, > 0 to enable GC, undefined defaults to disabled. */
     readonly gcFeature?: GCVersion;
+    /** Runtime version of the container when it was first created */
+    createContainerRuntimeVersion?: string;
+    /** Timestamp of creating the container */
```

Comment:
nit: Can this comment be consistent with the one for `createContainerRuntimeVersion`?